node/engine/messageservice/p2p-message-service/service.go

```diff
@@ -91,12 +91,11 @@ func NewMessageService(ip string, port int, me types.Address, pk []byte, useMdns
 	ms.checkError(err)
 
 	ms.key = messageKey
-
 	options := []libp2p.Option{
 		libp2p.Identity(messageKey),
-		libp2p.ListenAddrStrings(fmt.Sprintf("/ip4/%s/tcp/%d", ip, port)),
+		libp2p.ListenAddrStrings(fmt.Sprintf("/ip4/%s/tcp/%d", "0.0.0.0", port)),
```

Comment:
Is there a reason we're switching to `0.0.0.0`? Is it because specifying the IP address doesn't really matter?

Reply:
I'm changing this back to read from the `ip` argument instead of hardcoding to `0.0.0.0`. I can't tell why but the `0.0.0.0` argument seems to cause frequent failures in the CI. I will look into this further when addressing this issue: https://github.com/statechannels/go-nitro/issues/1489